src/backend/aspen/workflows/nextstrain_run/builds_templates/mega_template.yaml

```diff
@@ -15,7 +15,7 @@ builds:
     division: {division}
     location: {location}
     subsampling_scheme: {tree_type}
-    title: '{tree_type} tree for samples collected in {location}, {division}, {country}'  # not sure how this works w country-level builds
+    title: '{tree_type} tree for samples collected in {location}'  # not sure how this works w country-level builds
```

Comment:
will yaml builder confuse this `{location}` with the `{location}` 2 lines above? Love this set up to populate the title regardless!

Reply:
I can see how this looks really confusing. The short story is that it won't because we expand *this* string very early in our yaml generation process, and leave essentially all other `{somevar}` strings alone. But I'll try to figure out how to make this clearer.